docs/advanced/full-node.md

```diff
@@ -0,0 +1,75 @@
+# <img class="dcr-icon" src="/img/dcr-icons/AtoB.svg" /> Running a Full Node
+
+---
+
+This guide explains everything you need to know about running a full node on Decred.
+
+## What is a full node?
+
+A full node is a program that fully validates transactions and blocks, without having to rely on a third party. For this guide, we'll be using [dcrd](https://github.com/decred/dcrd), the most popular full node implementation for Decred. 
+
+## Why run a full node?
+
+There are a number of reasons to run a full node:
+
+- **Help the network:** Full nodes are the backbone of Decred. By validating and broadcasting transactions without relying on third parties&mdash; as lightweight nodes do&mdash;they help keep the network decentralized. Each full node added to the network adds to its security and robustness. 
+- **Safety & Privacy:** Because a lightweight node cannot fully validate transactions and blocks, it must trust third parties to do so. This means that the third party will have access to information about your transactions, as well as any personal data you give to that third party. For instance, many lightweight wallets rely on a centralized server to process transactions. Even Simplified Payment Verification (SPV) wallets, which typically connect directly to full nodes in the network, still rely on those other full nodes to process and broadcast their transactions. 
+
+
```

Comment:
Inconsistent number of blank lines here and elsewhere.  I see a mix of 1, 2, and 3 blank lines.

Reply:
Fixed